store/postgres/migrations/2018-12-21-003727_create_attribute_indexing_procedure/down.sql

```diff
@@ -0,0 +1,2 @@
+--  DROP ATTRIBUTE INDEXING FUNCTION
+DROP FUNCTION BuildSubgraphEntityAttributeIndexes(attribute_array entity_attribute_type[]);
```

Comment:
The name `BuildSubgraphEntityAttributeIndexes` doesn't appear anywhere else. Is this supposed to be `build_attribute_index` maybe?